src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(value) {
+      this.result += value;
+    },
+    subtract(value) {
+      this.result -= value;
+    },
+    divide(value) {
+      this.result /= value;
+    },
+    multiply(value) {
+      this.result *= value;
+    },
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
```

Comment:
To improve the readability of the code, we can add line break after each method.